### ISSUE TITLE
Paros pressure potentially showing temperature dependence (first order fit stuff)

### ISSUE THREAD

**Issue by dehann** [tjcrone/flowmow#13]
![bottom01_depth_temp](https://user-images.githubusercontent.com/6412556/44471686-6fbf1180-a5fa-11e8-9b36-996db755abd3.png)

Hi @tjcrone , looks like first order does not quite fit everything exactly.  I want to better understand the mismatch during the progression of the first on bottom segment (segment around -1480m on the first dive).  I plotted temperature next to the nav/paros estimates and think there might be correlation there.  Find [code at end of this notebook](https://github.com/tjcrone/flowmow/blob/nav/leverarm/vertveltest/notebooks/nav_spike_leverarm.ipynb).

As a next step, We are interested in the high frequency data from the paros --  
- [ ] 'de-trending' of paros data to the nav data below some reasonable frequency might be good enough;
- [ ] We should consider doing a sanity check using a seawater toolbox (or equivalent) to see if the paros is really measuring the change in temperature or something else is going on.
- [ ] I wonder if tidal cycles has anything to do with this -- what I mean is that the tidal change might be bringing colder water over the study site? 
- [ ] Do we know how Sentry nav converts pressure to depth ... are they doing temperature compensation?

I will continue to dig to find answers.

**Comment by dehann** [comment]
As a follow-on question, do you know where the vent site is in the temperature data?  Is there a place where we can see the temperature jump as the sensor goes through warmer water.

**Comment by tjcrone** [comment]
My first question is, where did you get your fitting parameters? And why is the first on -1? I would not expect this parameter to be -1. What if we take all the data, selected using a depth range, and find the zeroth and first order fitting parameters using least squares?

**Comment by tjcrone** [comment]
I took a quick stab at something that does not force the first parameter to -1, and it looks like there is still a trend in this short section of data, suggesting that the Sentry nav may be folding in the temperature to its depth calculation. This would make sense, but I'm not convinced this is the right thing to do. The temperature varies quite a bit in the vicinity a hydrothermal field, but the pressure-depth conversion is a function of the density of the entire overlying water column, which is not measured on the SBE3 or with the Sentry temperature sensors. My sense is that temperature variations measured by any of the Sentry sensors or SBE3 is going to misrepresent depth variations if these are factored in to the calculation.

**Comment by tjcrone** [comment]
I *think* we have Sentry height off of the seafloor, from the ADCP I believe, which might be a better measure of the movement of the vehicle IRL, and it might be worth comparing the Paros to the height. Is this possible?

**Comment by tjcrone** [comment]
Yeah I forgot to consider the topography issue so that was a dumb suggestion on my part. However I do think the Paros is the right instrument to use, basically as is with a multiplier and offset that we glean from either a least-squares linear model (Ax+B) fit to the Sentry depth, a look at the CTD data and a more fancy approach integrating water column density, or even something very simple like the conversion suggested in the Paros manual. My sense is that any of these approaches is likely fine. Leaning toward option 1-2.
